src/Features/Core/Portable/PdbSourceDocument/PdbSourceDocumentLoaderService.cs

```diff
@@ -125,15 +126,17 @@ internal sealed class PdbSourceDocumentLoaderService : IPdbSourceDocumentLoaderS
             return null;
         }
 
-        private async Task<SourceFileInfo?> TryGetSourceLinkFileAsync(SourceDocument sourceDocument, Encoding encoding, TelemetryMessage telemetry, CancellationToken cancellationToken)
+        private async Task<SourceFileInfo?> TryGetSourceLinkFileAsync(SourceDocument sourceDocument, Encoding encoding, TelemetryMessage telemetry, bool useExtendedTimeout, CancellationToken cancellationToken)
         {
             if (sourceDocument.SourceLinkUrl is null || _sourceLinkService.Value is null)
                 return null;
 
+            var timeout = useExtendedTimeout ? EasedSourceLinkTimeout : SourceLinkTimeout;
```

Comment:
```suggestion
            var timeout = useExtendedTimeout ? ExtendedSourceLinkTimeout : SourceLinkTimeout;
```

To keep the names consistent?